### PR TITLE
Tiny cleanups

### PR DESCRIPTION
**Pull request by knaperek**

### CODE REVIEW

**Review by pfouque — APPROVED** [review]

File: django_fsm_log/admin.py [hunk 1]
```diff
@@ -30,6 +30,6 @@ def get_readonly_fields(self, request, obj=None):
         return self.fields
 
     def get_queryset(self, request):
-        return super(StateLogInline, self).get_queryset(
+        return super().get_queryset(
             request
-        ).order_by(F('timestamp').desc())
```
Comment (ticosax) [inline django_fsm_log/admin.py:35]: I'd rather keep it that way.

**Review by ticosax — COMMENTED** [review]
Thanks, I just have a preference to keep the ordering like it was. As `desc()` it's more authoritative and explicit than interpretating the minus sign `-` a the beginning of a string.

File: django_fsm_log/admin.py (hunk 1 above)
Comment (MRigal) [reply django_fsm_log/admin.py:35]: Agree with @ticosax . @knaperek might you consider adjust your PR which otherwise is fully legit?

Comment (knaperek) [reply django_fsm_log/admin.py:35]: Alright, done and rebased.

**Review by ticosax — APPROVED** [review]
Thanks a lot, please rebase it against the head branch

**Pull request merged**